pkg/aws/common.go

```diff
@@ -58,7 +58,7 @@ func ValidLabelAdmins() []string {
 
 // VersionLabels are the labels which are considered version labels
 func VersionLabels() []string {
-	return []string{label.AWSOperatorVersion, label.ClusterOperatorVersion, label.Release}
```

Comment:
For now we only allow users to change the release version label because the other version labels will be changed accordingly.